ejector/ejector_config.go

```diff
@@ -69,21 +69,19 @@ type EjectorConfig struct {
 
 	// The period at which to periodically attempt to finalize ejections that have been started.
 	EjectionFinalizationPeriod time.Duration
-}
 
-var _ config.VerifiableConfig = (*EjectorSecretConfig)(nil)
+	// The number of blocks to wait before using a reference block number for quorum.
```

Comment:
> using a reference block number for quorum.

Not understanding this comment fully. How does a specific quorum factor in here?

Reply:
> using a reference block number for quorum

I guess my point is that I don't see how the `EjectorConfig` is for _a quorum_. Isn't this the some configuration for all quorums?